README.md

```diff
@@ -24,8 +26,9 @@ function add(x, y) {
   return x + y
 }
 ```
-
--- YOUR ANSWER HERE --
+es5, es6 would be with fat arrow;
+no
+add(x,y)
```

Comment:
I'll accept this answer, but it does assume that you previously assigned x and y  to Number values